test/test_rule_set_creating_shorthand.rb

```diff
@@ -121,8 +123,8 @@ def test_combining_background_with_size_into_shorthand
 
   def test_combining_background_with_size_and_no_position_into_shorthand
     properties = {'background-image' => 'url(\'chess.png\')', 'background-color' => 'gray',
```

Comment:
prefer 2-space all the way
```suggestion
    properties = {
      'background-image' => 'url(\'chess.png\')', 'background-color' => 'gray',
```